### ISSUE TITLE
Pages are not linked.

### ISSUE THREAD

**Issue by JeanHuit** [Hardy007-sys/Appraisal-System#3]
- The pages are not linked.
  -- Kindly link them, Let there be some form of interactivity.
- Build a navigation bar or something for easier navigation.
- Using bootstrap is all good, but because you do not fully understand it, you could not put it to good use.
  -- Please complete the course on codeacademy  and truly understand the basics, stick to the foundations and be good at it.
  --Overall, I will say 5 out of 10, for a first timer.

**Comment by Hardy007-sys** [comment]
Yeah I planned on linking the pages after I am done with all the interfaces.
Thanks...It's good to know I am making progress.

**Comment by JeanHuit** [comment]
Pages are still not linked.

**Comment by Hardy007-sys** [comment]
https://github.com/Hardy007-sys/Responsive-Front-end-Design

I sent you this on the 26th of April.
Below is a screenshot.

**Comment by Hardy007-sys** [comment]
https://github.com/Hardy007-sys/Responsive-Front-end-Design

I just downloaded it onto my computer. All the pages are linked.
